### PR TITLE
Configure babel and webpack

### PR DESCRIPTION
**Pull request by Mchl**

### CODE REVIEW

File: ui/.babelrc [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+  "presets": [
+    ["env", {
+      "targets": {
+        "browsers": [">5%", "not ie <=8"]
```
Comment (Mchl) [inline ui/.babelrc:5]: Browsers with over 5% market share as per https://github.com/ai/browserslist

File: templates/index.html [hunk 2]
```diff
@@ -24,8 +24,9 @@
     window.kab_mode = {% if kab_user %}true{% else %}false{% endif %};
 </script>
 
+<script src="{{ url_for('static', filename='babel_polyfill.js') }}"></script>
```
Comment (Mchl) [inline templates/index.html:27]: Not required just yet. Will be needed for ES2015+ features

File: ui/src/main.js [hunk 3]
```diff
@@ -0,0 +1,181 @@
+// Nav buttons
```
Comment (Mchl) [inline ui/src/main.js:1]: Copied as is from ./static

File: ui/src/stuff.js [hunk 4]
```diff
@@ -0,0 +1,18 @@
+const disqus_config = function () {
```
Comment (Mchl) [inline ui/src/stuff.js:1]: Introduced new syntax to showcase babel+webpack

File: ui/src/stuff.js [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+    this.page.url = 'http://gd77.sp5drs.xyz';
+    this.page.identifier = 'gd77';
+};
+
+(function(window, d) {
+    window.dataLayer = window.dataLayer || [];
+    const gtag = () => {
+      dataLayer.push(arguments);
+    }
+    gtag('js', (new Date()).toString());
```
Comment (Mchl) [inline ui/src/stuff.js:11]: What's the purpose of this?

File: ui/.babelrc (hunk 1 above)
Comment (Mchl) [reply ui/.babelrc:5]: Maybe `>1% in alt-eu`
http://browserl.ist/?q=%3E1%25+in+alt-eu

File: ui/src/stuff.js (hunk 5 above)
Comment (satanowski) [reply ui/src/stuff.js:11]: Ask Google Analytics, it's their code :)

Comment (Mchl) [reply ui/src/stuff.js:11]: Point me to source?

File: templates/index.html (hunk 2 above)
Comment (Mchl) [reply templates/index.html:27]: Actually it polyfills `Array.from` so we do need it.

**Comment by Mchl** [comment]
;(

**Pull request merged**